### ISSUE TITLE
[Feature Request] configure query timeout via command line options

### ISSUE THREAD

**Issue by tylerlong** [sosedoff/pgweb#319]
Current the query timeout is hard-coded: https://github.com/sosedoff/pgweb/blob/master/static/js/app.js#L57

The feature request is for adding the following command line option:

```
--query-timeout=    Set database query timeout in seconds (300)
```

**Comment by varp** [comment]
There is adjacent issue #287. @sosedoff I think you can close my one - #287

**Comment by varp** [comment]
I've added disable query timeout checkbox in UI. Look at PR https://github.com/sosedoff/pgweb/pull/339

**Comment by jmgtan** [comment]
Any updates on this one? Looks like PR #339 was closed but not merged.

**Comment by sosedoff** [comment]
Available in 0.12.0 via `--query-timeout` flag
